deployments/examples/gang/cmd/gangDeploy.sh

```diff
@@ -26,46 +26,23 @@ set -o pipefail
 JOBAMOUNT=$1
 GANGMEMBER=$2
 RUNTIMESEC=$3
```

Comment:
We really need to check and fail if there are not enough parameters.
We also should check if the parameters are integers with a simple check:
```
if [ "$1" -eq "$1" ] 2>/dev/null
then
    JOBAMOUNT=$1
else    
    echo "ERROR: first parameter must be an integer."
    exit 1
fi
```

Reply:
Sure, done.